.github/workflows/ci.yml

```diff
@@ -88,3 +88,42 @@ jobs:
       - "lint"
       - "build"
       - "pylint"
+  publish_gh:
```

Comment:
not sure if it's better a big file with all the steps (lint, test, release and deploy), or two files, one for the common tasks (lint & test), and another for release and deploy that depends on and it's only run for tags branches.
I'm talking about readability, becuase with `if: "startsWith(github.ref, 'refs/tags/v')"` you already covers the point

The other approach could have a (maybe) more explicit code at the top:
```yaml
on:
  push:
    tags: [ 'v*.*.*' ]
```

Reply:
The typical process today involves us manually applying a tag. If someone applies a tag from the CLI the `on: push: tags:` part will apply no matter what the tag is.